packages/store-engine-fs/src/index.ts

```diff
@@ -106,11 +106,7 @@ export class FileEngine implements CRUDEngine {
       }
 
       const filePath = this.resolvePath(tableName, primaryKey);
-      let newEntity: EntityType | string = entity;
-
-      if (typeof entity === 'object') {
-        newEntity = JSON.stringify(entity);
-      }
+      const newEntity: string = JSON.stringify(entity);
```

Comment:
i dont see a reason to make this change personally.

Reply:
it will lead to a quoted string in the unhappy path. we should use the previous method instead i believe. unless there is something im missing